src/NuGet.Core/NuGet.Commands/PushRunner.cs

```diff
@@ -32,15 +34,19 @@ public static class PushRunner
             PackageUpdateResource packageUpdateResource = await CommandRunnerUtility.GetPackageUpdateResource(sourceProvider, source);
 
             // only push to SymbolSource when the actual package is being pushed to the official NuGet.org
-            string symbolsSource = string.Empty;
-
             Uri sourceUri = packageUpdateResource.SourceUri;
-            if (!noSymbols && !sourceUri.IsFile && sourceUri.IsAbsoluteUri)
+            if (string.IsNullOrEmpty(symbolsSource) && !noSymbols && !sourceUri.IsFile && sourceUri.IsAbsoluteUri)
             {
                 if (sourceUri.Host.Equals(NuGetConstants.NuGetHostName, StringComparison.OrdinalIgnoreCase) // e.g. nuget.org
                     || sourceUri.Host.EndsWith("." + NuGetConstants.NuGetHostName, StringComparison.OrdinalIgnoreCase)) // *.nuget.org, e.g. www.nuget.org
                 {
                     symbolsSource = NuGetConstants.DefaultSymbolServerUrl;
+
+                    if (string.IsNullOrEmpty(symbolsApiKey))
+                    {
+                        // Use the nuget.org API key if it was given
```

Comment:
Actually, right here it is nuget.org. Look at the previous "if" statement where is compares against NuGetConstants.NuGetHostName

---
In reply to: [68473722](https://github.com/NuGet/NuGet.Client/pull/712#discussion_r68473722) [](ancestors = 68473722)